arbstate/inbox.go

```diff
@@ -63,6 +63,11 @@ const maxZeroheavyDecompressedLen = 101*MaxDecompressedLen/100 + 64
 const MaxSegmentsPerSequencerMessage = 100 * 1024
 const MinLifetimeSecondsForDataAvailabilityCert = 7 * 24 * 60 * 60 // one week
 
+var (
+	ErrNoBlobReader          = errors.New("blob batch payload was encountered but no BlobReader was configured")
+	ErrInvalidBlobDataFormat = errors.New("blob batch data is not a list of hashes as expected")
```

Comment:
I think instead of whitelisting these errors, we should return early from the fuzzer if the batch starts with a byte that IsDataAuthenticatedHeaderByte, as those batches shouldn't be arbitrary and are authenticated by the sequencer inbox

Reply:
Done.